website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -25,44 +44,103 @@ yarn global add @graphprotocol/graph-cli
 npm install -g @graphprotocol/graph-cli
 ```
 
-## Create your Subgraph in Subgraph Studio
+## Create your Subgraph
+
+### Create an Account
+
+Before deploying your subgraph you need to create a subgraph in [Subgraph Studio](https://thegraph.com/studio/).
```

Comment:
Did you mean
```suggestion
Before deploying your subgraph you need to create an account in [Subgraph Studio](https://thegraph.com/studio/).
```
? That would match the heading, but maybe you really did mean "create a subgraph", in which case I would rewrite as:

```suggestion
The first step is to create a subgraph in [Subgraph Studio](https://thegraph.com/studio/).
```